.travis.yml

```diff
@@ -2,6 +2,9 @@ language: ruby
 rvm:
   - 2.3.0
 script:
+  - bin/rake db:create RAILS_ENV=test
   - bin/rake db:migrate RAILS_ENV=test
```

Comment:
This is missing the `git submodule update` stuff. I wonder if we should be adding it to `bin/setup`and just using that in place of manual setup?